src/components/card/card.js

```diff
@@ -8,7 +8,8 @@
 angular.module('material.components.card', [
     'material.core'
   ])
-  .directive('mdCard', mdCardDirective);
+  .directive('mdCard', mdCardDirective)
+  .directive('mdPlainImage', mdPlainImageDirective);
```

Comment:
or perhaps we should just create a directive:

`<md-card-image ng-href=""></md-card-image>`